members/admin.py

```diff
@@ -88,17 +89,53 @@ def __init__(self, *args, **kwargs):
         if 'email' in self.fields:
             self.fields['email'].required = True
 
+class EmailFilter(SimpleListFilter):
+    title = 'MIT'
+    parameter_name = 'MIT'
+
+    def lookups(self, request, model_admin):
+        return [("", "None"), ("@mit.edu", "MIT"), ("@harvard.edu", "Harvard")]
```

Comment:
should be careful with harvard emails- they're formatted weirdly. i.e. @physics.harvard.edu, @g.harvard.edu... maybe remove the @ for the most generality? i think even ugrads are @college.harvard.edu

Reply:
Can update to remove @ sign and just keep domain